src/main/java/org/prebid/server/spring/config/bidder/postprocessor/BidderConfigurationBeanPostProcessor.java

```diff
@@ -0,0 +1,38 @@
+package org.prebid.server.spring.config.bidder.postprocessor;
+
+import org.apache.commons.lang3.ObjectUtils;
+import org.prebid.server.spring.config.bidder.model.BidderConfigurationProperties;
+import org.prebid.server.spring.config.bidder.model.CommonBidderConfigurationProperties;
+import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.config.BeanPostProcessor;
+
+import java.util.Objects;
+
+public class BidderConfigurationBeanPostProcessor implements BeanPostProcessor {
+
+    private final CommonBidderConfigurationProperties commonProperties;
+
+    public BidderConfigurationBeanPostProcessor(CommonBidderConfigurationProperties commonProperties) {
+        this.commonProperties = Objects.requireNonNull(commonProperties);
+    }
+
+    @Override
+    public Object postProcessAfterInitialization(Object bean, String beanName) throws BeansException {
+        if (bean instanceof BidderConfigurationProperties) {
+            final BidderConfigurationProperties properties = (BidderConfigurationProperties) bean;
+
+            properties.setEnabled(ObjectUtils.defaultIfNull(properties.getEnabled(), commonProperties.getEnabled()));
```

Comment:
Pls consider to use common method to set properties:
```
private <T, V> void setProperty(T target, Function<T, V> getter, BiConsumer<T, V> setter) {
    final V value = getter.apply(target);
    if (value == null) {
        @SuppressWarnings("unchecked") final V defaultValue = getter.apply((T) defaultProperties);
        setter.accept(target, defaultValue);
    }
}
```
Not critical. Up to you.